lib/client/ipam.go

```diff
@@ -589,22 +655,17 @@ func (c ipams) RemoveIPAMHost(host *string) error {
 	// Determine the hostname to use.
 	hostname := decideHostname(host)
 
-	// Release host affinities.
```

Comment:
I thought it did more than just that - I thought it reconfigured the allocation block to remove the host affinity.